### PR TITLE
[PROTOTYPE] Fixes #35 - Elided pagination for large result sets

### LINKED ISSUE #35: Pagination issue for large number of results
Searches with a large number of results and the Recent Entries page list every page of results, which gives a list of pages several lines long.

image

### PR DESCRIPTION
**Pull request by mhalverson**
So far I've verified that this implementation works with the `/events` page (see attached screenshot). Seeking feedback on the implementation before I verify that it works for the other pages.
screen shot 2018-10-18 at 11 35 59 pm

### CODE REVIEW

File: palanaeum/templates/palanaeum/pagination_nav.html [hunk 1]
```diff
@@ -2,9 +2,11 @@
     {% if page.has_previous %}
         <a class="w3-bar-item" href="{{ url }}?page={{ page.previous_page_number }}&{{ page_params }}"><span class="fa fa-chevron-left"></span></a>
     {% endif %}
-    {% for page_num in paginator.page_range %}
+    {% for page_num in page_numbers_to_show %}
         {% if page_num == page.number %}
             <a href="#" class="button1-accent w3-bar-item">{{ page_num }}</a>
+	{% elif page_num == -1 %}
```
[the remark this answers is not in the document]
Comment (mhalverson) [reply palanaeum/templates/palanaeum/pagination_nav.html]: Oops, my mistake with tabs vs spaces. I'm a longtime Emacs user still getting used to Vim, and it slipped through the cracks.

**Comment by mhalverson** [comment]
Will fix indentation and will remove TODO comments.

**Comment by m-strzelczyk** [comment]
OK, just make a rebase, because I merged some other changes to the master branch and I can merge yours then :)

**Comment by mhalverson** [comment]
rebase is done!

**Pull request merged**